### DISCUSSION TITLE
Question on verifying organization website

### DISCUSSION THREAD

**Discussion by aschrijver** [community/community#23722] [New to GitHub]
<p>I have a question on <a href="https://docs.github.com/en/free-pro-team@latest/github/setting-up-and-managing-organizations-and-teams/verifying-your-organizations-domain" rel="noopener nofollow ugc">Verifying your organization’s domain</a>.</p>
<p>My provider (a good one) does not allow me to create DNS TXT records that end in a dot, as the one suggested in the ‘Verify Domain’ screen in organization settings:</p>
<blockquote>
<p><code>_github-challenge-example.example.org.</code></p>
</blockquote>
<p>So I tried without the dot. Usually a new DNS record takes minutes to be available. I waited hours and still cannot validate. I know sometimes the changes can take longer time to be available. But if the dot is required it will never validate at all. Did I configure wrong with:</p>
<blockquote>
<p><strong>TXT record</strong>:</p>
<ul>
<li>name: <code>_github-challenge-example.example.org</code>
</li>
<li>value: <code>&lt;what-was-generated-by-github&gt;</code>
</li>
</ul>
</blockquote>
<p>The <code>dig</code> command seems to indicate that the TXT entry exists:</p>
<blockquote>
<p><code>; &lt;&lt;&gt;&gt; DiG 9.11.2-1ubuntu1.10-Ubuntu &lt;&lt;&gt;&gt; _github-challenge-example.example.org +nostats +nocomments +nocmd TXT</code><br>
<code>;; global options: +cmd</code><br>
<code>;_github-challenge-example.example.org. IN TXT</code></p>
</blockquote>

**Comment by otter-computer** [comment]
<p>The dot isn’t required—it’s usually appended automatically by your DNS provider.</p>
<p>From the looks of your examples it <em>could</em> be that you’ve added this DNS record to the subdomain <code>_github-challenge-example.example.org</code>. This will actually result in the full domain <code>_github-challenge-example.example.org.example.org</code>.</p>
<p>I’d check for a TXT record on the domain <code>_github-challenge-example.example.org.example.org</code> (replacing with your actual domain of course!) to see if the verification shows up there. If it does then you’ll want to edit the DNS record’s subdomain to remove your organisations domain from the end.</p>

**Comment by aschrijver** [comment]
<p>Thank you <a class="mention" href="/thomasshaped">@thomasshaped</a></p>
<p>I tried setting the TXT as per your suggestion (and my own domain filled in_, removing the other entry.</p>
<aside class="quote group-github-employees" data-username="thomasshaped" data-post="2" data-topic="137169">
<div class="title">
<div class="quote-controls"></div>
 thomasshaped:</div>
<blockquote>
<p><code>_github-challenge-example.example.org.example.org</code></p>
</blockquote>
</aside>
<p>But unfortunately it still does not validate.</p>

**Comment by otter-computer** [comment, marked as answer]
<p>Hi <a class="mention" href="/aschrijver">@aschrijver</a>, I don’t think I made my example clear, sorry! The subdomain you need to attach this TXT record to is only <code>_github-challenge-example</code>, without any <code>example.org</code> on the end.</p>
<p>If you’re still having trouble with this would you be able to post a link to the org you’re attempting to verify a domain for? That should help a little with the debugging.</p>

**Comment by aschrijver** [comment]
<p>Ah, that makes perfect sense, and… it worked :slight_smile:</p>
<p>I think the way the page describes what should be added to DNS could be more clearly explained, i.e. something like:</p>
<blockquote>
<p>Add the dns TXT record named  <code>_github-challenge-example</code> with the value generated below.</p>
</blockquote>
<p>Thanks a lot for your help <a class="mention" href="/thomasshaped">@thomasshaped</a>!</p>

**Comment by otter-computer** [comment]
<p>Amazing, sorry I didn’t make it clear enough first time, but glad it’s fixed now!</p>
<p>Yeah, I think so too. We have a feedback issue open about this page already so I’ll pass that on there for you :v:</p>
